pkg/comp-functions/functions/vshnmariadb/mariadb_deploy.go

```diff
@@ -0,0 +1,263 @@
+package vshnmariadb
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	xfnproto "github.com/crossplane/function-sdk-go/proto/v1beta1"
+	xhelmbeta1 "github.com/vshn/appcat/v4/apis/helm/release/v1beta1"
+	vshnv1 "github.com/vshn/appcat/v4/apis/vshn/v1"
+	"github.com/vshn/appcat/v4/pkg/common/utils"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/functions/common"
+	"github.com/vshn/appcat/v4/pkg/comp-functions/runtime"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	SLIBucketName = "vshn-test-bucket-for-sli"
+)
+
+// DeployMariadb will add deploy the objects to deploy mariadb
+func DeployMariadb(ctx context.Context, svc *runtime.ServiceRuntime) *xfnproto.Result {
+
+	l := controllerruntime.LoggerFrom(ctx)
```

Comment:
I haven't migrated all other code, but `svc` also provides a log now. `svc.Log`.

It currently doesn't matter which you use, they provide the same, but in the future we could add more service specific context to the logger like the claimname and namespace.

Reply:
Thanks for the hint. Adjusted the code accordingly